src/integrationTest/java/uk/gov/hmcts/darts/testutils/PostgresIntegrationBase.java

```diff
@@ -50,7 +50,9 @@ static void configureProperties(DynamicPropertyRegistry registry) {
 
     @BeforeEach
     void clearDb() {
+        dartsDatabase.resetSequences();
         dartsDatabase.clearDatabaseInThisOrder();
+        dartsDatabase.resetTablesWithPredefinedTestData();
```

Comment:
Author note: Aligning the `PostgresIntegrationBase` setup logic with `IntegrationBase`